plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/ModelLayerRenderer.kt

```diff
@@ -25,6 +25,7 @@ internal class ModelLayerRenderer(
   override fun initializeComponents(style: StyleInterface) {
     this.style = style
     source.bindTo(style)
+    setModelOpacity(locationModelLayerOptions.modelOpacity)
```

Comment:
I'm not convinced this is the right place. Suggestions welcomed.

Reply:
perhaps inside `addLayers` here?